apps/vaporgui/RenderHolder.h

```diff
@@ -19,6 +19,16 @@ namespace VAPoR {
 	class ParamsMgr;
 }
 
+class QCheckBoxTest : public QCheckBox {
+    Q_OBJECT;
+    
+public:
+    QCheckBoxTest(QString s):QCheckBox(s){}
+    ~QCheckBoxTest() {
+        printf("Test::Destructor() +++++++++\n");
```

Comment:
Please remove this print statement.

Reply:
1. That is correct but this was the simplest solution.
2. Well some datasets are just slower I guess. I am currently rendering a 549*349*32 dataset which is 4 million cell larger in under a second. I don't see this as a reason to remove the option to render curvilinear datasets entirely.
3. This print statement was to confirm that Qt does in fact delete the object, as per your other question. I'll do that.

Reply:
Interesting, I didn't know the performance would be that much different. 128^3 is probably the smallest resolution that's still useful to an user. My 128^3 data set is that sphere data set, essentially regular. Does the regular nature hurt performance of the curvi algorithm?

Reply:
No it doesn't make a difference if the grid is actually curvilinear or not. I don't know why you are experiencing worse performance. I can take a look at it though.